microsetta_interface/templates/reports.jinja2

```diff
@@ -90,9 +90,19 @@
                 <div class="barcode-col barcode-text">
                     {{ sample.sample_barcode }}
                 </div>
-                <div class="barcode-col barcode-col-end" id="btn-view-{{ sample.sample_id }}">
-                    <span class="sample-label">{{ _('Not Received Yet') }}</span>
-                </div>
+                {% if sample.sample_latest_scan_timestamp is none %}
+                    <div class="barcode-col barcode-col-end" id="btn-view-{{ sample.sample_id }}">
+                        <span class="sample-label">{{ _('Not Yet Received ') }} </span>
+                    </div>
+                {% elif sample.sample_latest_sample_information_update is none or sample.sample_latest_scan_timestamp > sample.sample_latest_sample_information_update %}
```

Comment:
@ayobi I just tested this on our staging server and I think we're running into a timezone issue somewhere. Here's what I did:
1) Claim a sample under a user account and fill out the sample collection info (sample date/time don't really matter as long as there's a valid value there).
2) Log into microsetta-admin as an admin user, hit Scan Barcode, find the sample, and mark it sample-has-inconsistencies
3) Log back into the UI as the normal user.

At this point, the sample should show as 'Sample Received - Information Needed,' but it's rolling past this condition and hitting 'Sample Received - Update Pending.' When I inspected the database, the `latest_sample_information_update` value in `ag.ag_kit_barcodes` was `2024-02-06 17:16:48.179585-08` and the `scan_timestamp` value in `barcodes.barcode_scans` was `2024-02-06 09:19:35.086172-08`.

The most expedient solution is probably to switch the insert/update value for `latest_sample_information_update` from `utcnow()` to `now()`. Can you see if changing that in the API resolves the issue, and ensure it doesn't introduce any unintended behavior?

Reply:
Ah ok, I changed it to now() and is committed on the api [PR](https://github.com/biocore/microsetta-private-api/pull/558/commits/b82a7579ee385dd19f81de6f5e5c398d8db0d4d0). I tested it in microsetta-admin and behaved as expected.